components/cam/bld/configure

```diff
@@ -1008,9 +983,6 @@ elsif ($phys_pkg =~ /ideal|adiabatic/) {$macrophys_pkg = 'none';}
 if ($clubb_sgs == 1) {
   $macrophys_pkg = 'clubb_sgs';
 }
-if ($shoc_sgs == 1) {
```

Comment:
just noting locations where shoc/p3 might be removed.

Reply:
all lines/code blocks that have shoc_sgs and P3 need to be retained